File: dna/templates/dna_template.py
name = "dna"

def read():
    case = input().strip()
    return case

def logic(case):
    # dna: the string of DNA bases, consisting of letters A, T, C, and G
    dna = case
    
    # TODO: YOUR CODE HERE
    result = ""
    for base in dna:
        if base == "A":
            result += "T"
        elif base == "T":
            result += "A"
        elif base == "C":
            result += "G"
        elif base == "G":
            result += "C"
    return result

def main():
    T = int(input().strip())
    for _ in range(T):
        result = logic(read())
        print(result)

if __name__ == "__main__":
    main()

File: dna/templates/DnaTemplate.java
// name: dna
import java.io.*;

public class DnaGuide {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    static class TestCase {
        String dna;
    }
    
    public static TestCase read() throws java.lang.Exception {
        TestCase testCase = new TestCase();
        String dna = br.readLine();
        testCase.dna = dna;
        return testCase;
    }

    public static void logic(TestCase testCase) {
        // dna: the string of DNA bases, consisting of letters A, T, C, and G
        String dna = testCase.dna;

        // TODO: YOUR CODE HERE

        return;
    }

	public static void main(String[] args) throws java.lang.Exception {
        int T = Integer.parseInt(br.readLine());
        for (int i = 0; i < T; i++) {
            logic(read());
        }
    }
}